### ISSUE TITLE
Date Picker Highlighting

### ISSUE THREAD

**Issue by tcasey** [mominsamir/smDateTimeRangePicker#86]
When setting a date value to the `ng-model` the date picker does not show it as selected but the value in the input field is the correct value. For example if I've set thg `ng-model` to `25-December-2016` the input field will show `25-December-2016` and the date picker itself will show the `moment()` value (ex: the date of your browser).

**Comment by tcasey** [comment]
After some investigation I found that updating the `currentDate` from within the `PickerCtrl.prototype.configureNgModel` allowed me to update the `date-time-header` however I'm still having issues with updating the `sm-calender`. From what I've seen so far it seems that because my application has to make a couple API calls to get the `ng-model` value the directive loads up before it's on the `DOM` and never updates the `initialDate` or `currentDate` with that value. Thus leading to a date picker that doesn't highlight. (you can replicate this by applying a `setTimeout` to the `ng-model` value) 

I'm going to look into this a little bit more and send in a PR if I can figure it out. Any help or direction would be greatly appreciated!

**Comment by devlemire** [comment]
Nerd.

**Comment by mominsamir** [comment]
duplicate of #76

**Comment by mominsamir** [comment]
I think this requires some modification in calender ctrl to reflect manually updated date .

**Comment by tcasey** [comment]
No, this is a different issue with the date picker. I have made a fix for it and will make a pull request.

**Comment by mominsamir** [comment]
sure
